Xamarin.Forms.Core/FontImageSource.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.Forms.Internals;
+using ValidateValueDelegate = Xamarin.Forms.BindableProperty.ValidateValueDelegate;
+
+namespace Xamarin.Forms
+{
+	public partial class FontImageSource : ImageSource
+	{
+		private static BindableProperty CreateBindableProperty<T>(
```

Comment:
I'd avoid trying to be smart here. it makes the code harder to understand in the big picture